modules/oadp-s3-compatible-backup-storage-providers.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc
+
+:_content-type: CONCEPT
+[id="oadp-s3-compatible-backup-storage-providers_{context}"]
+= AWS S3 compatible backup storage providers
+
+OADP is compatible with many object storage providers for use with different backup and snapshot operations. Several object storage providers are fully supported, several are unsupported but known to work, and some have known  limitations.
+
+== AWS S3 providers for backup storage locations
```

Comment:
You need an anchor ID for the heading on line 11, 13, 29, and 42

Reply:
+1. All headings need anchors, as per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings.

Reply:
@JoeAldinger @stevsmit I added the anchors. Thank you for your help!